src/convertToObject.js

```diff
@@ -7,6 +7,26 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  // eslint-disable-next-line no-debugger
+  if (!sourceString) {
+    return {};
+  }
+
+  const styleToObject = {};
+  const partsOfExpresions = sourceString.split(';');
+
+  partsOfExpresions.forEach((part) => {
+    const index = part.indexOf(':');
+
+    const name = part.substring(0, index).trim();
+    const value = part.substring(index + 1).trim();
+
+    if (name && value) {
+      styleToObject[name] = value;
+    }
+  });
+
+  return styleToObject;
```

Comment:
The function 'convertToObject' is well-named and clearly describes its purpose. However, the code could benefit from chaining methods for better readability and conciseness, as long as it does not involve modifying an existing array or object.